doc/tdshim_spec.md

```diff
@@ -640,11 +645,10 @@ Table 3.4-1 shows the E820 memory map.
 
 | E820 Memory Type       | Value | Usage                                                                 | TD-Memory Type | OS Action                          |
 |:-----------------------|:------|:----------------------------------------------------------------------|:---------------|:-----------------------------------|
-| AddressRangeMemory     | 1     | Usable by OS.                                                         | Private        | Use directly                       |
+| AddressRangeMemory     | 1     | Usable by OS.                                                         | Private/Unaccepted | Use directly                       |
```

Comment:
please use `Private (Accepted or Unaccepted)`.

OS Action `For Accepted, use directly. For Unaccepted, use after convert to private page.`